deployer/src/lib.rs

```diff
@@ -44,7 +44,9 @@ pub async fn start(
 
     persistence.cleanup_invalid_states().await.unwrap();
 
-    let runnable_deployments = persistence.get_all_runnable_deployments().await.unwrap();
+    let mut runnable_deployments = persistence.get_all_runnable_deployments().await.unwrap();
+    runnable_deployments.reverse();
```

Comment:
Nit: would it be cheaper to rather change the order in the query? Although this list will likely be small, and it only runs on start-up, so it probably doesn't really matter.

Reply:
Oh, we might want to add a comment about why we reverse, will make it quicker for people out-of-the loop to understand the reasoning.